java/connector-node/risingwave-connector-service/src/main/java/com/risingwave/connector/source/common/PostgresValidator.java

```diff
@@ -540,7 +540,9 @@ private boolean isDataTypeCompatible(String pgDataType, Data.DataType.TypeName t
                 return val == Data.DataType.TypeName.DOUBLE_VALUE;
             case "decimal":
             case "numeric":
-                return val == Data.DataType.TypeName.DECIMAL_VALUE;
+                return val == Data.DataType.TypeName.DECIMAL_VALUE
+                        || val == Data.DataType.TypeName.INT256_VALUE
```

Comment:
Please add some comment for the additional type mapping.

Reply:
added